src/content/blam/tags/bitmap/readme.md

```diff
@@ -192,15 +187,17 @@ _NOTE: Tool will sometimes split a sequence across several pages. This does not
 _QUIRK: Only the start of a sequence divider needs to be the color defined on the color plate. After that it can be any color, or could even just end prematurely._
 
 ### Dummy Space
-[Images still need to be uploaded] Space usage on a sprite with dummy space vs without
+<img src="media/bitmap_sprite_dummy_space.png" width="400">
+
+Space usage on a sprite with dummy space vs without
 
 Dummy space is space that is counted toward the size and position of your sprite, but not included in the data of the sprite in the bitmap. This is useful for two reasons: It allows a sprite that's smaller than other sprites in a sequence, and it tells tool how to center the smaller sprite without bulking up the file size with padding. Dummy space must be a solid color that is different from the sprite border and sequence separator colors.
 
 ### Sprite Usage
 Under "... more sprite processing" there is a drop down box that controls the background color of the texture page. This is used to avoid color bleeding in from outside the sprite boundaries at runtime and creating outline-like artifacts.
-* **blend/add/subtract/max**: Makes the background black.
-* **multiply/min**: Makes the background white.
-* **double multiply**: Makes the background grey (50% grey).
+ - **blend/add/subtract/max**: Makes the background black.
+ - **multiply/min**: Makes the background white.
+ - **double multiply**: Makes the background grey (50% grey).
```

Comment:
Minor, but would you be OK removing these single space indents? I think it would play better with editors set up for a more usual 2 space indent.

Reply:
Part of this was mainly readability in source form. But I am not super attached to the indents